dcpy/connectors/edm/recipes.py

```diff
@@ -77,3 +118,171 @@ def import_recipe(
         con.execute(update(recipes_table).values(data_library_version=version))
 
         con.commit()
+
+        if import_table_name is not None:
+            logger.info(f"Renaming table {recipe_name} to {import_table_name}")
+            con.execute(
+                text(f"ALTER TABLE {recipe_name} RENAME TO {import_table_name};")
+            )
+            con.commit()
+
+
+def _recipe_datasets(recipe):
+    return recipe.get("inputs", {}).get("datasets", []) or []
```

Comment:
nit, but is the `or` redundant with the two `get`s?

Reply:
@fvankrieken somewhat frustratingly not, in the case when the value is set to None (as opposed to the key being missing)
```
recipe = {"inputs": {"datasets": None}}
recipe.get("inputs", {}).get("datasets", []) # -> None
```

Reply:
Ohh, gotcha